src/docfx/Errors.cs

```diff
@@ -112,6 +112,12 @@ public static Error GitNotFound()
         public static Error NullValue(Range range, string name)
             => new Error(ErrorLevel.Info, "null-value", $"{range} '{name}' contains null value", line: range.StartLine, column: range.StartCharacter);
 
+        public static Error UnknownFieldType(Range range, string propName, string typeName)
+            => new Error(ErrorLevel.Warning, "unknown-field-type", $"{range} Could not find member '{propName}' on object of type '{typeName}'", line: range.StartLine, column: range.StartCharacter);
```

Comment:
>object [](start = 115, length = 6)

is it possible to print the name of the object, or even the path of the object relative to root?

Reply:
Good point, we can build the path and put it into warning.
The name of the object, do you mean the Yaml/json file name?